certbot-apache/certbot_apache/configurator.py

```diff
@@ -150,14 +152,14 @@ def __init__(self, *args, **kwargs):
         super(ApacheConfigurator, self).__init__(*args, **kwargs)
 
         # Add name_server association dict
-        self.assoc = dict()
+        self.assoc = dict()  # type: Dict[str, obj.VirtualHost]
         # Outstanding challenges
-        self._chall_out = set()
+        self._chall_out = set()  # type: Set[Tuple[KeyAuthorizationAnnotatedChallenge, ...]]
```

Comment:
Due to enumerate(iterable) used later on self._chall_out in another function, it appears that a list may be valid as well. In this case the type could be Set[Union[List[KeyAuthorizationAnnotatedChallenge], Tuple[KeyAuthorizationAnnotatedChallenge, ...]]]. However, the automated tests use a tuple. And the aforementioned type annotation is about 109 characters by itself, which leads to issues passing pylint. I don't think there is a valid way to break up that type annotation to pass pylint and be a valid type annotation.

Reply:
You're right. Thank you.